__tests__/unit/integration/api.test.js

```diff
@@ -0,0 +1,148 @@
+const { appendToFile } = require("../../../routes/api");
+const fs = require("fs");
+const app = require("../../../app");
+const request = require("supertest");
+
+describe("appendToFile()", () => {
+  test("adds to a file", function () {
+    let testFile = fs.writeFile("./facts-dump/test.txt", "", (err) => {
+      if (err) throw err;
+    });
+    let writeToTestFile = appendToFile("./facts-dump/test.txt", "hello world");
+    fs.readFile("./facts-dump/test.txt", "utf8", (err, data) => {
+      if (err) throw err;
+
+      expect(data).toEqual("hello world");
+    });
+  });
+});
+
+describe("routes", () => {
+  // done() pattern needed for all tests
+  // without done() we get a jest TCPSERVERWRAP error
+
+  // test for route "/:num(-?[0-9]+)"
```

Comment:
you say this, but none of your tests use `done` :) using `async` / `await` should make `done` unnecessary. can you remove this comment?